feature/gnoi/system/tests/copying_debug_files_test/copying_debug_files_test.go

```diff
@@ -31,10 +31,11 @@ var (
 		ondatra.NOKIA:   "sr_bgp_mgr",
 		ondatra.ARISTA:  "IpRib",
 		ondatra.JUNIPER: "rpd",
+		ondatra.CISCO:   "bgp",
 	}
 	components = map[ondatra.Vendor]string{
 		ondatra.ARISTA:  "Chassis",
-		ondatra.CISCO:   "Chassis",
+		ondatra.CISCO:   "Rack 0",
```

Comment:
This is good enough.   It would be better if the component tree was fetched and the code searches for a component with type "CHASSIS" and reads the name.  This would eliminate the need to hard code various strings here for each vendor.